packages/client-purpose-updater/tsconfig.check.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+      "noEmit": true,
+    },
+    "include": ["src"]
```

Comment:
```suggestion
    "include": ["src", "test"]
```

Reply:
Still, let's add the tests folder to the TS config check file, to make sure the tests folder is covered when it is added